app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -515,4 +515,50 @@ class GithubWebhook extends RequestHandler<Body> {
       return null;
     }
   }
+
+  /// Returns true if the changes to [file] are all code comments.
+  ///
+  /// If that cannot be determined with confidence, returns false. False
+  /// negatives (e.g., for /* */-style multi-line comments) should be expected.
+  bool _allChangesAreCodeComments(PullRequestFile file) {
+    final int? linesAdded = file.additionsCount;
+    final int? linesDeleted = file.deletionsCount;
+    final String? patch = file.patch;
+    // If information is missing, err or the side of assuming it's a non-comment
+    // change.
+    if (linesAdded == null || linesDeleted == null || patch == null) {
+      return false;
+    }
+
+    // Ensure that the file is a language reconized by the check below.
+    const Set<String> codeExtensions = <String>{
+      'cc',
+      'cpp',
+      'dart',
+      'java',
+      'kt',
+      'm',
+      'mm',
+      'swift',
+    };
+    final String filename = file.filename!;
+    final String? extension = filename.contains('.') ?
+      filename.split('.').last.toLowerCase() : null;
+    if (extension == null || !codeExtensions.contains(extension)) {
+      return false;
+    }
+
+    // Only handle single-line comments; identifying multi-line comments
+    // would require the full file and non-trivial parsing.
```

Comment:
This also doesn't notice changes that only affect the commend side of a line with code and a trailing comment (which is perfectly fine, just saying we might want to mention that here in case someone is confused about it).

Reply:
Good point, added that to the comment.